### PR TITLE
Create cfe-partners.php

### PR DESCRIPTION
**Pull request by cdavis-ubc**

### CODE REVIEW

File: cfe-partners.php [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+
+$the_title          = get_the_title();
+
+// Some custom fields
+$location			= get_post_meta( absint( $post_id ), 'Location', true );
+$description		= get_post_meta( absint( $post_id ), 'Description', true );
+$session 	    	= get_post_meta( absint( $post_id ), 'Session', true );
+$website            = get_post_meta( absint( $post_id ), 'Website', true );
+$theme              = get_post_meta( absint( $post_id ), 'Theme', true );
+$address            = get_post_meta( absint( $post_id ), 'Physical Address', true );
```
Comment (richardtape) [inline cfe-partners.php]: A post meta key with a space? That doesn't seem right? Post meta keys are almost always lowercase and have `_` or `-` to separate them

File: cfe-partners.php [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+              	
+              	<div class="row-fluid">
+                 
+                  	<div class="span12">
+                   
+                    	<div class="modal-body-content">
+							
+							<div id="community-partner-entry">
+								<img class="pull-right alignright" src="<?php echo esc_url( $post_thumbnail ); ?>" />
+								<a href="<?php echo esc_attr( $website ); ?>" target="_blank"><?php echo esc_html( $website ); ?></a><strong>Theme:</strong> <?php echo esc_html( $theme ); ?>
```
Comment (richardtape) [inline cfe-partners.php]: `href` attribute should use `esc_url()`

**Comment by cdavis-ubc** [comment]
esc_url added line 86

**Comment by richardtape** [comment]
OK this, and the 2 from within the last hour are now on verf. :+1:

**Pull request merged**